File: Python Code/Daily_Flash_Code_0/Week12/11Apr/71-DailyFlash_Solutions/11_April_Solution_One/C/Program0.c
#include<stdio.h>

void main(){

	int n = 0;

	printf("Enter Number of Elements : ");
	scanf("%d",&n);

	int arr1[n];
	int arr2[n];
	
	printf("Enter For array 1\n");
	for(int i=0; i<n; i++){

		scanf("%d",&arr1[i]);
	}

	printf("Enter For array 2\n");
	for(int i=0; i<n; i++){

		scanf("%d",&arr2[i]);
	}

	int min1 = arr1[0];
	int min2 = arr1[0];

	for(int i=0; i<n; i++){
			
		if(arr1[i] > min1){
			min2 = min1;
			min1 = arr1[i];
		}
		else if(arr1[i] > min2){
			min2 = arr1[i];
		}

		if(arr2[i] > min1){
			min2 = min1;
			min1 = arr2[i];
		}
		else if(arr2[i] > min2){
			min2 = arr2[i];
		}
	
	}

	printf("Second Largest Number %d\n",min2);

}

File: Python Code/Daily_Flash_Code_0/Week12/11Apr/71-DailyFlash_Solutions/11_April_Solution_One/Java/Program0.java
import java.io.*;
import java.util.*;

class ArrayOps{

	public static void main(String ... args)throws IOException{

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		System.out.println("Enter No of Elements : ");
		int m = Integer.parseInt(br.readLine());

		int arr1[] = new int[m];
		int arr2[] = new int[m];


	System.out.println("Enter Elements For Array 1");
	for(int i=0; i<m; i++){

		arr1[i] = Integer.parseInt(br.readLine());
	}

	System.out.println("Enter Elements For Array 2");
	for(int i=0; i<m; i++){

		arr2[i] = Integer.parseInt(br.readLine());
	}

	int min1 = arr1[0];
	int min2 = arr1[0];

	for(int i=0; i<m; i++){
			
		if(arr1[i] > min1){
			min2 = min1;
			min1 = arr1[i];
		}
		else if(arr1[i] > min2){
			min2 = arr1[i];
		}

		if(arr2[i] > min1){
			min2 = min1;
			min1 = arr2[i];
		}
		else if(arr2[i] > min2){
			min2 = arr2[i];
		}
	
	}

	System.out.printf("Second Largest Number %d\n",min2);
	
	}
}
